build_tools/python_deploy/iree-meta-package/README.md

```diff
@@ -0,0 +1,5 @@
+# IREE meta package
+
+Meta package to install compatible versions of
+ * `iree-base-compiler`
+ * `iree-base-runtime`
```

Comment:
Since this README will appear on PyPI, we could link to https://pypi.org/project/iree-base-compiler/ and https://pypi.org/project/iree-base-runtime/. Could see if there are other meta packages on PyPI and what they do.

Another option is to use the main project README here. Maybe we could start with this text about the meta package and then append the rest of the common README at the bottom...?